### PR TITLE
Minor fixes (proposal)

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

**Comment by KamranMackey** [comment]
@johnnybenson you should merge this. :)

**Comment by bhtaw293** [comment]
Hello

**Comment by Asif181188** [comment]
Hi

**Comment by Asif181188** [comment]
Dhaka

File: api-license-agreement.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@ provisions herein, please contact us at [api@tumblr.com].
      Firehose, to manipulate information and Content from the Tumblr Firehose for
      the purposes of non-public analysis and search; and (d) use and display the
      Tumblr Marks only to identify that the Licensed Materials originate from the
-     Tumblr Services.Licensee Applications do not include those applications that
+     Tumblr Services. Licensee Applications do not include those applications that
```
Comment (mibo420) [inline api-license-agreement.txt]: ????

**Comment by oudakyo0** [comment]
Great job!!!!
Let's keep being GREAT !
775-419-2484

CALL
DAK


anytime
I love r / t FAMILY

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request closed without merging**